pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -196,7 +196,7 @@ func (m *managerImpl) Start(diskInfoProvider DiskInfoProvider, podFunc ActivePod
 	go func() {
 		for {
 			if evictedPods := m.synchronize(diskInfoProvider, podFunc); evictedPods != nil {
-				klog.Infof("eviction manager: pods %s evicted, waiting for pod to be cleaned up", format.Pods(evictedPods))
+				klog.InfoS("Eviction manager: pods evicted, waiting for pod to be cleaned up", "pods", format.Pods(evictedPods))
```

Comment:
is it expected that on line 168 the `klog.KObj(attrs.Pod)` used and here is `format.Pods` is used?

Reply:
I want to use klog.KObj(attrs.Pod), But I think evictedPods(multiple pods) is not work here,so I keep it not change.

Reply:
@serathius what should we be doing in the case of multiple pods like this? Do you think this is ok?

Reply:
I think we should migrate it to `klog.KObj` to support Json and make changes in formatting easier. We didn't add specific methods for multiple pods as we weren't sure how often this case will came up. 

This is the first case using, so for now let's just implement a simple for loop.